From: Modestas Vainius <modax@debian.org>
Subject: disable INSTALL_RPATH_USE_LINK_PATH where it triggers useless RPATH
 .
 cantor-backend-r: lintian output: 'binary-or-shlib-defines-rpath ./usr/bin/cantor_rserver /usr/lib64/R/lib', automatically rejected package.
 cantor-backend-r: lintian output: 'binary-or-shlib-defines-rpath ./usr/lib/kde4/cantor_rbackend.so /usr/lib64/R/lib', automatically rejected package.
 kalzium: lintian output: 'binary-or-shlib-defines-rpath ./usr/bin/kalzium /usr/lib/ocaml', automatically rejected package.
Forwarded: not-needed
Origin: vendor
Last-Update: 2011-07-09
---
 cantor/src/backends/R/CMakeLists.txt         |    1 +
 cantor/src/backends/R/rserver/CMakeLists.txt |    1 +
 kalzium/compoundviewer/CMakeLists.txt        |    3 ++-
 kalzium/src/CMakeLists.txt                   |    1 +
 4 files changed, 5 insertions(+), 1 deletion(-)

--- a/cantor/src/backends/R/rserver/CMakeLists.txt
+++ b/cantor/src/backends/R/rserver/CMakeLists.txt
@@ -17,6 +17,7 @@ add_custom_target(renvvars COMMAND ${R_E
 add_custom_target(rautoloads COMMAND ${R_EXECUTABLE} --slave --file=${CMAKE_CURRENT_SOURCE_DIR}/tools/autoloads.r > ${CMAKE_CURRENT_BINARY_DIR}/rautoloads.h )
 
 kde4_add_executable( cantor_rserver ${RServer_SRCS} )
+set_target_properties( cantor_rserver PROPERTIES INSTALL_RPATH_USE_LINK_PATH false)
 target_link_libraries( cantor_rserver ${KDE4_KDEUI_LIBS} cantorlibs ${R_LIBRARIES} ${KDE4_KIO_LIBS})
 add_dependencies(cantor_rserver renvvars rautoloads)
 
--- a/kalzium/src/CMakeLists.txt
+++ b/kalzium/src/CMakeLists.txt
@@ -130,6 +130,7 @@ kde4_add_app_icon(kalzium_SRCS "${CMAKE_
 
 kde4_add_executable(kalzium ${kalzium_SRCS})
 
+set_target_properties(kalzium PROPERTIES INSTALL_RPATH_USE_LINK_PATH false)
 target_link_libraries(kalzium ${KDE4_KHTML_LIBRARY} ${KDE4_KNEWSTUFF3_LIBS} ${KDE4_KUNITCONVERSION_LIBS} science)
 
 if (OPENBABEL2_FOUND)
--- a/cantor/src/backends/R/CMakeLists.txt
+++ b/cantor/src/backends/R/CMakeLists.txt
@@ -21,6 +21,7 @@ QT4_ADD_DBUS_INTERFACE(RBackend_SRCS ${n
 kde4_add_ui_files(RBackend_SRCS settings.ui)
 
 kde4_add_plugin( cantor_rbackend ${RBackend_SRCS} )
+set_target_properties( cantor_rbackend PROPERTIES INSTALL_RPATH_USE_LINK_PATH false)
 target_link_libraries( cantor_rbackend ${KDE4_KDEUI_LIBS} ${KDE4_KIO_LIBS} cantorlibs ${R_USED_LIBS})
 
 install( FILES rbackend.desktop  DESTINATION ${SERVICES_INSTALL_DIR}/cantor)
--- a/kalzium/compoundviewer/CMakeLists.txt
+++ b/kalzium/compoundviewer/CMakeLists.txt
@@ -22,6 +22,7 @@ target_link_libraries(compoundviewer
    avogadro
 )
 
-set_target_properties(compoundviewer PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION})
+set_target_properties(compoundviewer PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION}
+   INSTALL_RPATH_USE_LINK_PATH false)
 
 install(TARGETS compoundviewer ${INSTALL_TARGETS_DEFAULT_ARGS})
